ui/src/pages/Logs/Logs.tsx

```diff
@@ -16,7 +16,7 @@ import {
 } from '@mui/material';
 import SearchIcon from '@mui/icons-material/Search';
 import * as React from 'react';
-import { SideBar } from '../../components/Sidebar';
+import { SideBar } from '../../components/Sidebar/Sidebar';
```

Comment:
Could you make the SideBar component the default export to stay consistent with our other components?